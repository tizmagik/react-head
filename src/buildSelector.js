const buildSelector = obj =>
  Object.keys(obj)
    .map(k => `[${k}="${obj[k]}"]`)
    .join('');

export default buildSelector;
