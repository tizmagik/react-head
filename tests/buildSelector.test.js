import buildSelector from '../src/buildSelector';

describe('buildSelector', () => {
  it('builds selector based on given object', () => {
    expect(
      buildSelector({
        one: 'first',
        two: 'second',
        three: 'third',
      })
    ).toMatchSnapshot();
  });
});
