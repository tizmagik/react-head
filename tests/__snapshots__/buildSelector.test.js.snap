// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buildSelector builds selector based on given object 1`] = `"[one=\\"first\\"][two=\\"second\\"][three=\\"third\\"]"`;
